pages/edge/tutorials/cdn-on-wasmer-edge.mdx

```diff
@@ -0,0 +1,357 @@
+import { Steps, Callout, FileTree } from "nextra-theme-docs";
+import StaticSite from "@components/deploy/quickstart/StaticSite";
+import edgeReactImg from "@assets/deploy/tutorials/edge-react.png";
+import Install from "@components/install.mdx";
+import Login from "@components/login.mdx";
+import CliVersionCallout from "@components/deploy/CliVersionCallout.mdx";
+
+# Creating a CDN on Wasmer Edge
+
+In this tutorial, we will create a CDN on Wasmer Edge. We'll be using the [`static-web-server`]()
+
+## Prerequisites
+
+The project requires the following tools to be installed on your system:
+
+- [Rust](https://www.rust-lang.org/tools/install)
+
+## Deploying a Static Site
+
+<Steps>
+
+<Install />
+
+<CliVersionCallout />
+
+<Login />
+
+### Initialize the project
+
+We'll initialize the project with the `static-site` template.
+
+Navigate to the directory where you want to create the project and run the following command:
+
+```bash
+$ wasmer app create
+```
+
+The above command will give you the following prompt:
+
+```bash
+App type:
+> Static website # Select Static website
+  HTTP server
+  Browser shell
+  JS Worker (experimental)
+  Python Worker
+Who should own this package?: wasmer
+No package found or specified.
+What should the package be called? It will be published under wasmer: <your-cdn-name> # This will be the package name of your CDN
+What should be the name of the app? <NAME>.wasmer.app: <your-cdn-name> # This will be the name of your CDN
+Would you like to publish the app now? no # Select no for now
+Writing app config to '/Volumes/Work/Projects/wasmer-edge/wasmer-edge-cdn/app.yaml'
+To (re)deploy your app, run 'wasmer deploy'
+```
+
+Select the above configuration mentioned in the comments and press enter.
+
+The above command will initialize a blank project with the following structure.
+
+<FileTree>
+  <FileTree.Folder name="your-top-level-dir" defaultOpen>
+    <FileTree.Folder name="public" defaultOpen>
+      <FileTree.File name="index.html" />
+    </FileTree.Folder>
+    <FileTree.File name="app.yaml" />
+    <FileTree.File name="wasmer.toml" />
+  </FileTree.Folder>
+</FileTree>
+
+### Changing configuration for CDN deployment
+
+Let's first delete the `index.html` file from the `public` directory.
+
+```bash
+$ rm public/index.html
+```
+
+Now as you might notice, this project uses the `static-web-server` package which is a static web server written in Rust.
+
+Most notably, it supports changing the compression level, cache control, cors, and more.
+All these configurations can be super useful when deploying a CDN.
+
+So, let's add a `config.toml` file in the `config` directory.
+
+```bash
+$ mkdir config
+$ touch config/config.toml
+```
+
+You can copy the following `config.toml`.
+
+```toml
+[general]
+
```

Comment:
We don't need to showcase every detail here, nor that big of a config. Please summarize it to the essential things only